views/home.tmpl

```diff
@@ -79,10 +79,10 @@
                                     {{range .Blocks}}
                                     <tr data-height="{{.Height}}" data-link-class="fs18">
                                         <td class="text-left pl-1" data-type="height"><a href="/block/{{.Height}}" class="fs18">{{.Height}}</a></td>
-                                        <td class="text-center" data-type="tx">{{.Transactions}}</td>
-                                        <td class="text-center" data-type="votes">{{.Voters}}</td>
-                                        <td class="text-center" data-type="tickets">{{.FreshStake}}</td>
-                                        <td class="text-center" data-type="revocations">{{.Revocations}}</td>
+                                        <td class="text-right" data-type="tx">{{.Transactions}}</td>
+                                        <td class="text-right" data-type="votes">{{.Voters}}</td>
+                                        <td class="text-right" data-type="tickets">{{.FreshStake}}</td>
+                                        <td class="text-right" data-type="revocations">{{.Revocations}}</td>
```

Comment:
The home page looks OK, however I'm not sure it's clearly better.  Also note the large new white space in the Transactions column here too.

Before:

![image](https://user-images.githubusercontent.com/9373513/78581529-9b0b6a00-77f9-11ea-9bca-2884873f2be2.png)


This PR:

![image](https://user-images.githubusercontent.com/9373513/78581429-7e6f3200-77f9-11ea-96b5-4e2158d3d840.png)

Reply:
I agree. I decided against right aligning those columns simply because it looks terrible.